api/schema/imageBuilder.yaml

```diff
@@ -214,21 +214,6 @@ paths:
             example: '123e4567-e89b-12d3-a456-426655440000'
           required: true
           description: Id of compose to get the clones of
-        - in: query
-          name: limit
-          schema:
-            type: integer
-            default: 100
-            minimum: 1
-            maximum: 100
-          description: max amount of clones, default 100
-        - in: query
-          name: offset
-          schema:
-            type: integer
-            default: 0
-            minimum: 0
-          description: clones page offset, default 0
```

Comment:
The fact that the clone endpoint is having troubles now is because of the change here: https://github.com/RedHatInsights/image-builder-frontend/pull/1366/files#diff-bd7250abbffed5537d68df905da2a5693d884dbfaa60889e01129f86d1df316eR8-R23 Basically you shouldn't endup in a situation where you need to change the API representation. If it happens it's because something else is wrong :slightly_smiling_face: